moviebot/controller/server.py

```diff
@@ -55,80 +54,12 @@ def set_webhook():
 
 
 def run(config):
-    """Runs execute_agent in ControllerMessenger and starts flask server.
+    """Runs execute_agent in ControllerTelegram and starts flask server.
 
     Args:
         config: agent settings
 
     """
     controller_telegram.execute_agent(config)
-    controller_messenger.execute_agent(config)
-    # Messenger verify token
-    verify_token()
     set_webhook()
     app.run(host="0.0.0.0", port=environ.get("PORT", 5000))
-
-
-def verify_token():
-    """Gets verify token from config file."""
-    path = "config/bot_token.yaml"
-    with open(path, "r") as file:
-        config = yaml.load(file, Loader=yaml.Loader)
-        VERIFY_TOKEN = config["MESSENGER_VERIFY_TOKEN"]
-        return VERIFY_TOKEN
-
-
-@app.route("/", methods=["GET", "POST"])
-def receive_message():
```

Comment:
Shouldn't this method stay for the Telegram controller?

Reply:
According to the description, this method is for receiving Messenger POST requests.